docs/configuration.md

```diff
@@ -32,7 +32,7 @@ JSON:
 }
 ```
 
-JS(ESM):
+JS (ESM):
```

Comment:
```suggestion
JS (ES Module)
```

Reply:
Good idea, added this real quick!

Reply:
@fisker The long form of `ESM` is more commonly `ES Modules` (with `s`) - what do you think about that instead?

Reply:
I don't think so, the config is a single module, why `s`? I'm a native English speaker, I'll let other maintainer decide.

Reply:
This is what I was thinking was your reasoning for it, and technically,  it kind of makes sense

However, in the context of JS, I would argue it's more common for people to look for the project module format (not a description of a single file) - JS module formats are often described as "CommonJS" vs "ES Modules"

Also, people may search for the more common search term "ES Modules" and not be able to find this part of the page